statsmodels/tsa/statespace/tests/results/results_structural.py

```diff
@@ -164,11 +164,21 @@
 seasonal = {
     'models': [{'irregular': True, 'seasonal': 4}],
     'params': [38.1704278, 0.1],
-    'llf': -655.3337155,
-    'kwargs': {},
+    'llf': -678.8138005,
+    'kwargs': {'loglikelihood_burn': 0},
     'rtol': 1e-6
 }
 
+freq_seasonal = {
```

Comment:
This test case is now against the KFAS results in test_ucm.R instead of just a regression test.